File: programmers/05.문자열 내 p와 y의 개수.js
function solution(s){
    //let answer = true;

    let p = s.toUpperCase().split("P").length;
    // console.log(p);
    // console.log(p.length);

    
    let y = s.toUpperCase().split("Y").length;
    // console.log(y);
    // console.log(y.length);


    return p === y;
}

console.log(solution("ySSppyysdppyYp"));
console.log(solution("yoypopp"));

File: programmers/java/05. 문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        boolean answer = true;
        
        int p = 0;
        int y = 0;
        
        for(int i=0; i< s.length(); i++) {  //charAt(i) : i번째(인덱스) 
            if(s.charAt(i) == 'P' || s.charAt(i) == 'p') {
                p++;
            } else if(s.charAt(i) == 'Y' || s.charAt(i) == 'y') {
                y++;
            } 
        }
        
        if(p != y) answer = false;
        
        return answer;
    }
}
